libraries/Adapters/Microsoft.Bot.Builder.Adapters.Twilio/TwilioEvent.cs

```diff
@@ -0,0 +1,44 @@
+﻿// Copyright(c) Microsoft Corporation.All rights reserved.
+// Licensed under the MIT License.
+
+namespace Microsoft.Bot.Builder.Adapters.Twilio
+{
+    public class TwilioEvent
```

Comment:
A lot of this properties don't seem to be used, will wait for the unit tests to understand the use for them.

Reply:
Ok, we will wait for the unit tests. Thanks.